1.9/api/access.md

```diff
@@ -0,0 +1,40 @@
+---
+post_title: Cluster Access
+nav_title: Cluster Access
+menu_order: 1
+---
+
+The cluster address may be obtained using the following methods:
```

Comment:
However, "host" in ULR parlance just includes the IP or hostname, not the port or scheme.